parcels/particlesets/particlesetsoa.py

```diff
@@ -187,6 +193,36 @@ def indexed_subset(self, indices):
         return ParticleCollectionIteratorSOA(self._collection,
                                              subset=indices)
 
+    def populate_indices(self):
```

Comment:
Do we need to also highlight this method in a tutorial or example? @reint-fischer, can you think of a good place to document this?